spec/yaml/swiftnav/sbp/orientation.yaml

```diff
@@ -10,7 +10,7 @@
 
 package: swiftnav.sbp.orientation
 description: Orientation Messages
-stable: False
+stable: True
```

Comment:
@reimerix Are these output messages stable in your opinion? I think `MSG_BASELINE_HEADING` is the only one from the GNSS engine and it hasn't needed updating in ages.

Reply:
I'm not sure if the Duro Inertial outputs these. The descriptions imply that it shouldn't, but I could have sworn it did. Either way I think it would be fine to deprecate these in the future if there's a need to output something equivalent but we don't think these definitions are acceptable.